model/atmosphere/dycore/tests/dycore_stencil_tests/test_apply_divergence_damping_and_update_vn.py

```diff
@@ -12,7 +12,9 @@
 
 import icon4py.model.common.type_alias as ta
 import icon4py.model.testing.helpers as test_helpers
-from icon4py.model.atmosphere.dycore.dycore_states import DivergenceDampingOrder
+from icon4py.model.atmosphere.dycore.dycore_states import (  # type: ignore[import-untyped]
```

Comment:
Why is this needed? Shouldn't all `icon4py` subpackages be typed (even if they are incomplete or partially wrong for now)? Do all subpackages contain the compulsory `py.typed` file?

This comment applies to all `ignore[import-untyped]` comments from icon4py in this PR

Reply:
> Why is this needed?

I wish I knew.

> Shouldn't all icon4py subpackages be typed (even if they are incomplete or partially wrong for now)?

No idea :stuck_out_tongue: 

> Do all subpackages contain the compulsory py.typed file?
I'll check.

Reply:
I've quickly checked and it seems that there are other packages besides `icon4py.model.testing` (which is being fixed in this PR) missing the `py.typed` file:

- icon4py.model.atmosphere.dycore
- icon4py.model.atmosphere.subgrid_scale_physics.microphysics
- icon4py.model.atmosphere.driver

Could you also add the `py.typed` files into their `src` folders and then removed the `type-ignore`?

Reply:
https://github.com/C2SM/icon4py/pull/743/commits/33b4ec8500dcccd4273808f7e2851d6ab62badaa

I added one to `model/driver/src/icon4py/model/driver`. Is that what you meant for icon4py.model.atmosphere.driver?

Reply:
Oops, yes, it was a typo from my side.